.github/CONTRIBUTOR_GUIDE/ct/AppName.md

```diff
@@ -0,0 +1,231 @@
+# **AppName<span></span>.sh Scripts**
+ `AppName.sh` scripts found in the `/ct` directory. These scripts are responsible for the installation of the desired application. For this guide we take `/ct/snipeit.sh` as example.
```

Comment:
Just out of curiosity. Why snipeit? Is this a very easy example?
I am asking, because a more popular application may create more engagement. I, at least, don't know "snipeit".

Reply:
It just happens to one of my one scripts, so i know it best^^. Also it is not to complicated/long and inlcudes almost all topics wich are in the guides. This can always be changed of course.